### PR TITLE
[WIP] Switch to isilon filesystem

### LINKED ISSUE #468: Use MAST Isilon directory as filesystem
As @bourque and I remembered yesterday, the `jwqladm` user on the virtual machine has access to the MAST Isilon directory, which contains more data than the copy we've been using up until now from Mary Romelfanger. Using the MAST directory as our filesystem would give our web app/monitors access to more data.

This change might just be as simple as updating the `filesystem` key in each of our personal `config.json` files.

If we do this, though, we'll need to be sure to integrate the ASB permissions-checking feature to make sure that we're not doing anything illicit (i.e. making preview images that live in unrestricted directories) with proprietary data.

### PR DESCRIPTION
**Pull request by bourque**
This PR makes appropriate changes to switch the `jwql` backend filesystem from a local copy on our central storage area to the storage for operations on isilon storage. 

Closes #468

### CODE REVIEW

**Comment by bourque** [comment]
@bhilbert4 @Johannes-Sahlmann Would one of you be able to review this PR (ideally sometime this week)?

**Comment by bhilbert4** [comment]
I'll try to give things a look during breaks in the python class I'm in this week.

File: jwql/tests/test_pipeline_tools.py [hunk 1]
```diff
@@ -29,8 +29,14 @@
 # Determine if tests are being run on jenkins
 ON_JENKINS = '/home/jenkins' in os.path.expanduser('~')
 
+# Determine if user has access to filesystem
+NO_FILESYSTEM_ACCESS = True
+if not ON_JENKINS:
+    NO_FILESYSTEM_ACCESS = not os.path.exists(get_config()['public_filesystem']) or not os.path.exists(get_config()['proprietary_filesystem'])
```
[the remark this answers is not in the document]
Comment (bourque) [reply jwql/tests/test_pipeline_tools.py]: Yes, I believe there is a scenario in which this applies, for example, if you are running the tests on your local machine and don't have access to the isilon filesystem (which I think is the current case for all JWQL developers -- only the admin account has access to isilon storage).

File: jwql/tests/test_pipeline_tools.py [hunk 2]
```diff
@@ -41,7 +47,7 @@ def test_completed_pipeline_steps():
         File to be checked
     """
 
-    filename = os.path.join(get_config()['filesystem'], 'jw00312',
+    filename = os.path.join(get_config()['proprietary_filesystem'], 'jw00312', 'jw00312002001',
```
Comment (bhilbert4) [inline jwql/tests/test_pipeline_tools.py]: Are we sure this file will always be in the proprietary filesystem? Could it migrate to the public filesystem eventually?

File: jwql/utils/monitor_template.py [hunk 3]
```diff
@@ -105,7 +105,7 @@ def monitor_template_main():
     #      'suffix': 'uncal'}
 
     # Example of locating a dataset in the filesystem
-    filesystem = get_config()['filesystem']
+    filesystem = get_config()['public_filesystem']
```
Comment (bhilbert4) [inline jwql/utils/monitor_template.py]: Might be worth mentioning the proprietary filesystem here as well just so developers don't forget to search it?

File: jwql/website/apps/jwql/data_containers.py [hunk 4]
```diff
@@ -732,10 +737,11 @@ def get_proposal_info(filepaths):
...
 
-        fits_search_filepath = os.path.join(
-            FILESYSTEM_DIR, 'jw{}'.format(proposal), 'jw{}*.fits'.format(proposal)
-        )
-        num_files.append(len(glob.glob(fits_search_filepath)))
+        fits_search_filepath_proprietary = os.path.join(PROPRIETARY_FILESYSTEM_DIR, 'jw{}'.format(proposal), '*', 'jw{}*.fits'.format(proposal))
+        fits_search_filepath_public = os.path.join(PUBLIC_FILESYSTEM_DIR, 'jw{}'.format(proposal), '*', 'jw{}*.fits'.format(proposal))
+        fits_files = glob.glob(fits_search_filepath_proprietary)
+        fits_files.extend(glob.glob(fits_search_filepath_public))
+        num_files.append(len(fits_files))
```
Comment (bhilbert4) [inline jwql/website/apps/jwql/data_containers.py]: Will there ever be files that appear in both public and proprietary filesystems at the same time?

**Comment by bourque** [comment]
This PR is quite old, and I am starting over again in #726, so I am closing this PR.

**Pull request closed without merging**